newIDE/app/src/Utils/GDevelopServices/User.js

```diff
@@ -66,3 +80,11 @@ export const getUsernameAvailability = (
     .get(`${GDevelopUserApi.baseUrl}/username-availability/${username}`)
     .then(response => response.data);
 };
+
+export const instagramLinkPrefix = 'https://instagram.com/';
+export const twitterLinkPrefix = 'https://twitter.com/';
+export const facebookLinkPrefix = 'https://facebook.com/';
+export const youtubeLinkPrefix = 'https://youtube.com/';
+export const tiktokLinkPrefix = 'https://tiktok.com/@';
```

Comment:
Do you strip the possible @ that a user may type in the field to avoid having a `@@Alexandre_` in the url?